apps/ledger-live-mobile/src/screens/AnalyticsOptInPrompt/variantA/Details.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { memo } from "react";
+import styled from "styled-components/native";
+import { Flex, IconsLegacy, Link, Text, Toggle } from "@ledgerhq/native-ui";
+import { TrackScreen } from "~/analytics";
+import { useTranslation } from "react-i18next";
+import { Check, Close } from "@ledgerhq/native-ui/assets/icons";
+import Button from "~/components/Button";
+import Switch from "~/components/Switch";
+
+const Container = styled(Flex).attrs({
+  paddingVertical: 20,
+  paddingHorizontal: 16,
+  flex: 1,
+  justifyContent: "space-between",
+})``;
+
+const View = styled(Flex).attrs({
+  width: "100%",
+})``;
+
+const Titles = styled(Flex).attrs({
+  py: 3,
+  width: "100%",
+})``;
+
+const Content = styled(Flex).attrs({
+  py: 6,
+  width: "100%",
+})``;
+
+const Bottom = styled(Flex).attrs({
+  paddingBottom: 7,
+  paddingTop: 0,
+  width: "100%",
+})``;
+
+const OptionContainer = styled(Flex).attrs({
+  width: "100%",
+  alignItems: "flex-start",
+})``;
+
+const OptionRow = styled(Flex).attrs({
+  flexDirection: "row",
+  justifyContent: "space-between",
+  alignItems: "center",
+  backgroundColor: "opacityDefault.c05",
+  borderRadius: 12,
+  width: "100%",
+  padding: 4,
+})``;
```

Comment:
We have a lot of styled components that are shared between both variant and components, can you reuse only one version please?